nexus/users/tests/test_models.py

```diff
@@ -8,14 +9,22 @@
 class UserModelTestCase(TestCase):
 
     def test_create_user(self):
-        u = User.objects.create_user(email='f@F.com', password='abc', first_name="F", last_name='B')
+        u = f.create_user(
```

Comment:
Since this is a model test case, we should actually use `User.objects.create` to trigger appropriate method.